test/unit/model/config-test.js

```diff
@@ -0,0 +1,28 @@
+import Model from 'controller/model';
+
+describe('Model', function() {
+    const config = {
+        file: '//playertest.longtailvideo.com/adaptive/bbbfull/bbbfull.m3u8',
+        image: 'http://d3el35u4qe4frz.cloudfront.net/bkaovAYt-480.jpg',
+        preload: 'auto'
+    };
+
+    let model;
+
+    before(function () {
+        model = new Model();
+        model.setup(config);
+    });
+
+    describe('return value', function() {
+        it.only('should return an object on setup', function() {
+            expect(model).to.be.an('object');
+        });
+    });
+
+    describe('state', function() {
+        it.only('should return an idle state', function() {
+            expect(model.get('state')).to.equal('idle');
```

Comment:
I'd put this test in the same describe block as the previous test. And reword the `it` to say something like that it is testing the default value of state. Another useful test would be to confirm that the default state is overwritten if passed in the setup config.